### PR TITLE
Display correct size values

### LINKED ISSUE #1083: inconsistant values for field `actual_num_blocks` between `zbox meta` and `zbox list`
lets say I have a file: `/raw.file_example_MP4_1920_18MG.mp4`, when I get the file details using: `zbox list`

`zbox list --allocation $alloc --remotepath /raw.file_example_MP4_1920_18MG.mp4 --json --silent | jq .`
```
[
  {
    "name": "raw.file_example_MP4_1920_18MG.mp4",
    "path": "/raw.file_example_MP4_1920_18MG.mp4",
    "type": "f",
    "size": 8471456,
    "hash": "9a7953230084f1c5b683623e60d900683bf1a026e99e3a628684a79f6f08fa91",
    "mimetype": "video/fmp4",
    "num_blocks": 130,
    "lookup_hash": "8b06c8f6bf817a97bbc2efbae77d6c3d07fea8e48604332fab57adb7d8ce10f1",
    "encryption_key": "",
    "actual_size": 4235728,
    "actual_num_blocks": 64,
    "created_at": 1688326508,
    "updated_at": 1688326508,
    "list": null,
    "chunk_size": 65536
  }
]

```

but for `zbox meta` I get a different value for `ActualNumBlocks`

`zbox meta --allocation $alloc --remotepath /raw.file_example_MP4_1920_18MG.mp4 --json --silent | jq .`

```
{
  "Name": "raw.file_example_MP4_1920_18MG.mp4",
  "Type": "f",
  "Path": "/raw.file_example_MP4_1920_18MG.mp4",
  "LookupHash": "8b06c8f6bf817a97bbc2efbae77d6c3d07fea8e48604332fab57adb7d8ce10f1",
  "Hash": "9a7953230084f1c5b683623e60d900683bf1a026e99e3a628684a79f6f08fa91",
  "MimeType": "video/fmp4",
  "Size": 4235728,
  "ActualFileSize": 4235728,
  "ActualNumBlocks": 65,
  "EncryptedKey": "",
  "Collaborators": null
}
```

### PR DESCRIPTION
**Pull request by din-mukhammed**
### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test: https://github.com/0chain/system_test/pull/726
- zboxcli: https://github.com/0chain/zboxcli/pull/456/
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | master  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

File: zboxcore/sdk/allocation.go [hunk 1]
```diff
@@ -2139,7 +2139,7 @@ func (a *Allocation) getConsensuses() (fullConsensus, consensusThreshold int) {
 		return a.DataShards, a.DataShards
 	}
 
-	return a.DataShards + a.ParityShards, a.DataShards + 1
+	return a.DataShards + a.ParityShards, a.DataShards
```
Comment (Kishan-Dhakan) [inline zboxcore/sdk/allocation.go]: What's the reason for this change?
Minimum Data+1 number of shards are required to recover the full data as far as I remember. @lpoli please review this change.

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: zboxcore/sdk/allocation.go (hunk 1 above)
Comment (din-mukhammed) [reply zboxcore/sdk/allocation.go]: I think consensus threshold (as in minimum number of shards needed to reconstruct the original data) should be equal to number of data shards, I don't quite understand why we use "data+1".

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | sprint-june-5  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | sprint-july-1  |
| 0chain  | staging  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Review by dabasov — COMMENTED** [review]
lgtm

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | sprint-july-1  |
| 0chain  | sprint-july-1  |
| blobber  | sprint-july-1  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | sprint-july-1  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | sprint-july-1  |
| 0chain  | sprint-july-1  |
| blobber  | sprint-july-1  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | sprint-july-1  |
| 0chain  | sprint-july-1  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  | 1033-fix-list  |
| gosdk  | fix/get-list-from-blobbers  |
| zboxcli  | display-correct-sizes  |
| zwalletcli  | sprint-july-1  |
| 0chain  | sprint-july-1  |
| blobber  | sprint-july-1  |
| authorizer  | staging  |
| 0box  | staging  |
| 0dns  | staging  |

**Pull request merged**